pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -15,6 +15,7 @@
   nixosTests,
   rocksdb,
   callPackage,
+  withFoundationdb ? false,
```

Comment:
as this is a breaking change i suggest adding a changelog item.

Reply:
Currently, foundationdb is only added to the nix closure, but the cargo feature isn't enabled. I would consider this a breaking change, but I can add a changelog entry is you insist.

Reply:
nvm, you are right, i thought it was enabled at some point.